docs/pages/ViewManager/VM-View-Templates.md

```diff
@@ -90,15 +90,15 @@ Then define the Default Field Separator, Custom Field, Custom Separator, and add
 
 Use the drop-down lists to filter view templates by Disciplines and View Types.
 
-![View Templates filter view templates](../../../)
+![View Templates filter view templates](../../../assets\images\ViewManager\VM-Vt-Filter.gif)
 
 <sub>Note: the version on the image may not reflect the [latest version of ViewManager/DiRootsOne](https://diroots.com/revit-plugins/dirootsone/).</sub>
 
 ### Quick visualization
 
 Visualize only the selected view templates, by clicking on Hide un-checked. You can also visualize only the view templates with associated views, by clicking on Has Associated Views.
```

Comment:
"Visualize only the selected View Templates, by clicking on "Hide Unchecked". You can also visualize only the View Templates with associated Views, by clicking on "Has Associated Views"."

Reply:
Fixed